apps/agreement/contracts/Agreement.sol

```diff
@@ -1158,10 +1160,11 @@ contract Agreement is IAgreement, AragonApp {
     /**
     * @dev Tell whether an action can be closed
     * @param _action Action instance to be queried
+    * @param _checkingFromDisputable Whether the check is being performed from the disputable itself
     * @return True if the action can be closed, false otherwise
     */
-    function _canClose(Action storage _action) internal view returns (bool) {
-        return _canProceed(_action) && _action.disputable.canClose(_action.disputableActionId);
+    function _canClose(Action storage _action, bool _checkingFromDisputable) internal view returns (bool) {
```

Comment:
Inside this function we already have the disputable, so I think we could get rid of `_checkingFromDisputable` param and check it inside. For the external view function it’s not a problem, actually the opposite, more consistent.